src/include/duckdb/main/capi/header_generation/apis/v1/v1.2/v1.2.0.json

```diff
@@ -134,6 +135,7 @@
         "duckdb_get_double",
         "duckdb_get_date",
         "duckdb_get_time",
+        "duckdb_get_time_ns",
```

Comment:
Adding new CAPI functions to an old version is not allowed. It would break backwards compatibility. New functions should first be added to https://github.com/duckdb/duckdb/tree/main/src/include/duckdb/main/capi/header_generation/apis/v1/unstable and will be stabilized later by adding them to a new vx.y.z json before a release